articles/white-hat-seo-black-hat-seo/index.md

```diff
@@ -0,0 +1,71 @@
+
+- [Google Analytics](https://analytics.google.com/analytics/web/)
+Google Analytics is a web analytics tool offered by Google that provides basic data and analysis regarding website traffic. Google Analytics is used to measure website activities such as session duration, pages per session, bounce rate, etc. It also supports real-time analytics, allowing a user to gain information about visitors who are actively on the site. One of the best Google Analytics features is that it can identify badly performing sites based on information such as where visitors came from, how long they remained on the website, and their geographical location. This helps the user to improve the website qualitatively based on the reports. More details about Google Analytics can be found [here](https://en.wikipedia.org/wiki/Google_Analytics).
+
+- [Google Search Console](https://search.google.com/search-console/welcome)
+Formerly known as &quot;Google webmaster tools&quot;, google search console is a free Google tool that allows you to build, maintain, and troubleshoot your website&#39;s visibility in Google Search results. It offers information on how Google crawls, indexes, and delivers web pages. This can assist website owners in monitoring and optimizing search performance. It helps you check the security issues with the website. This helps in avoiding malware attacks and site hacking. It also allows you to specify a preferred domain for your site, which determines how the site URL is displayed in SERPs. It helps to see what keyword searches on Google lead to the site being featured in the SERPs. Learn more about Google Search Console [here](https://en.wikipedia.org/wiki/Google_Search_Console) and [here](https://www.section.io/engineering-education/google-search-console-introduction/) is a simple tutorial on how to use the same.
+
+- [Yoast](https://yoast.com/)
+It is a WordPress SEO plug-in that was previously known as WordPress SEO. This, like the other SEO tools, is focused on keyword research. It is the most well-known SEO plugin, having been downloaded millions of times and running on millions of websites. Yoast SEO helps in optimizing your site&#39;s pages and articles for the specific keywords you want to rank for by defining a focus keyword. It ranks posts and articles based on the number of times the keywords were used, the number of paragraphs, the length of sentences, the use of internal links, and so on. Using this information, the plug-in analyzes the posts, indicating whether additional optimization is necessary or not. As a result, the readability of the posts improves, hence contributing to a large collection of unique content. [Here&#39;s](https://yoast.com/what-does-yoast-seo-do/) a detailed explanation of what Yoast SEO does.

```

Comment:
A good number of your sentences start with _it_. Try mixing up the beginnings of sentences to avoid repetition. I've made suggestions where I could and changed some as well. Backtrack through the article, specifically the _Tools_ section, and see if there are instances I missed that could use the variety.